src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+// elements constants
+const cellsArrow = document.querySelectorAll('.field-cell');
+const startButton = document.querySelector('.button');
+
+// change button
+startButton.addEventListener('click', () => {
+  if (startButton.classList.contains('start')) {
+    startButton.classList.replace('start', 'restart');
+    startButton.textContent = 'Restart';
+
+    getRandomCell();
+    getRandomCell();
+  } else {
+    // startButton.classList.replace('restart', 'start');
+    // startButton.textContent = 'Start';
+
+    cellsArrow.forEach(function(cell) {
+      cell.classList.remove('field-cell--2');
+      cell.classList.remove('field-cell--4');
+      cell.textContent = '';
+    });
+
+    getRandomCell();
+    getRandomCell();
+  }
+});
+
+// getRandomCell function
+function getRandomCell() {
+  const cellNumber = Math.floor(Math.random() * 16);
+
+  const cellValue = (function() {
```

Comment:
You can simplify this function using a ternary operator instead of an immediately invoked function expression (IIFE).